src/lender/adapters/mkr/deployer.sol

```diff
@@ -58,7 +44,7 @@ contract MKRLenderDeployer is LenderDeployer {
         mkrDeployer = address(1);
     }
 
-    function deploy() public {
+    function deployMkr() public {
```

Comment:
Was this because of overriding?

Reply:
It wasn't actually necessary, thought it was at first. Fixed now.